src/chain/ethereum.rs

```diff
@@ -57,6 +57,12 @@ pub enum EthereumTransactionParsingError {
     MissingMaxFeePerGas,
 }
 
+#[derive(Debug, Clone, PartialEq)]
+pub enum EthereumCalculateReceiptRootError {
+    /// [tx.type] is invalid.
+    InvalidType(u8),
```

Comment:
`EthereumTransactionParsingError::InvalidType` is self-described, but here it should be `InvalidTxType`.